easybuild/easyblocks/f/fsl.py

```diff
@@ -63,22 +66,48 @@ def configure_step(self):
         fslmachtype = out.strip()
         self.log.debug("FSL machine type: %s" % fslmachtype)
 
-        # prepare config
-        # either using matching config, or copy closest match
+        # Patch files for ver. < 5.0.10 patch multiple config directories
+        if LooseVersion(self.version) >= LooseVersion('5.0.10'):
+            # Check if a specific machine type directory is patched
+            systype_regex = re.compile("^diff.*config\/(.*(apple|gnu|i686|linux|spark)(?:(?!\/).)*)", re.M)
+
+            patched_cfgs = []
+            best_cfg = None
```

Comment:
@vanzod This needs to go outside of this `if` clause to ensure it's always defined for old FSL versions?

Reply:
True. Sloppy brain